query-parser/src/find.rs

```diff
@@ -136,3 +156,49 @@ pub fn parse_find(expr: edn::Value) -> QueryParseResult {
     }
     return Err(QueryParseError::InvalidInput(expr));
 }
+
+#[cfg(test)]
+mod test_parse {
+    extern crate edn;
+
+    use self::edn::types::{to_keyword, to_symbol};
+    use super::mentat_query::{
+        Element,
+        FindSpec,
+        SrcVar,
+        Variable,
+    };
+    use super::*;
+
+    // TODO: when #224 lands, fix to_keyword to be variadic.
```

Comment:
#224 has now landed, but I don't understand how you intend to make `to_keyword` variadic.  Even with `Into<>` tricks, you still need `None`...